cmip6_preprocessing/preprocessing.py

```diff
@@ -436,6 +422,8 @@ def combined_preprocessing(ds):
         ds = broadcast_lonlat(ds)
         # shift all lons to consistent 0-360
         ds = correct_lon(ds)
+        # codify units with pint
+        ds = ds.pint.quantify()
```

Comment:
Should we do this inside `correct_units`?

Reply:
That would fix your bug, but whether or not you want to conceptually combine "adding codified physical units to the data" and "potentially change what units the data is expressed in" is up to you.